### ISSUE TITLE
Enum cases with associated types are "invisible" when using Xcode 10 beta 5

### ISSUE THREAD

**Issue by stuartro** [krzysztofzablocki/Sourcery#661]
Code:
```swift
public enum Test {
   case a
   case b(Int)
}
```

Stencil:
```stencil
{% for enum in types.enums.all where enum.cases.all.count > 0 and not enum.accessLevel == 'private' %}
{{ enum.name }}:
  {% for case in enum.cases %}
    - {{ case.name}}
  {% endfor %}
{% endfor %}
```

Results (Xcode 9.4.1):
```
Test:
    - a
    - b
```

Results (Xcode 10 beta 5):
```
Test:
    - a
```

**Comment by marcelofabri** [comment]
Probably related to #639

**Comment by stuartro** [comment]
Seems to be working with Xcode 10 beta 6

**Comment by stuartro** [comment]
Correction. Not sure what's changed, but I'm now running into the same issue again with Xcode 10 beta 6.

**Comment by tcldr** [comment]
Yup – same here. Current master version still affected so closing #639 doesn't seem to have resolved.

**Comment by krzysztofzablocki** [comment]
@tcldr could you provide a sample test case that fails ? that would help

**Comment by tcldr** [comment]
All is well again. Must have been accidentally using an older version. Thanks.

**Comment by krzysztofzablocki** [comment]
Should be fixed in 0.14.0
